packages/components/src/TimeSlider.tsx

```diff
@@ -148,10 +155,13 @@ const PopOvers = (props: PopOversProps): JSX.Element => {
     startTime > endTime ? startTime : endTime
   );
 
-  useEffect(() => {
-    setFirstTime(startTime > endTime ? endTime : startTime);
-    setSecondTime(startTime > endTime ? startTime : endTime);
-  }, [startTime, endTime]);
+  useEffect(
+    function setTimeOnPropTimeChangee() {
```

Comment:
This one's on State change not Prop change, and typo with "Changee":
```suggestion
    function setTimeOnStateTimeChange() {
```